ufactory_xarm7/README.md

```diff
@@ -2,6 +2,14 @@
 
 Requires MuJoCo 2.3.3 or later.
 
+## Changelog
+
+- **17/12/2024**: Improved object grasping by:
+  - Adding two collision box meshes as pads for each finger.
+  - Setting `armature=0.1` for the joints.
+
+  These changes resolve [issue #83](https://github.com/google-deepmind/mujoco_menagerie/issues/83) and are implemented in [PR #131](https://github.com/google-deepmind/mujoco_menagerie/pull/131).
```

Comment:
I'd prefer removing this part if possible (the issue/PR part).

BTW, feel free to add your name next to "Improved object grasping" --> (thanks to @s1lent4gnt)

Reply:
I have correct it. Thanks!